webview/cocoa.py

```diff
@@ -253,7 +262,7 @@ def show(self):
             BrowserView.app.run()
 
     def destroy(self):
-        BrowserView.app.stop_(self)
+        PyObjCTools.AppHelper.callAfter(self.window.close)
```

Comment:
The current design makes no distinction between 'parent' or 'child' windows.
All windows, whether created by Python or JS, are 'children' of the app. You
want `pywebview` to give the initial window some special status? So that the
app terminates when it is closed? If this is the desired behaviour, it can be
implemented easily, I think. There is a `mainWindow` property of the app that
we can utilise. But the official documentation for the JS `window.open` makes
no such requirement. Try `window.open` in a browser, and the child window
stays open even after closing the browser's main window.